projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -37,6 +37,10 @@
       height: 32px;
     }
 
+    &.medium {
```

Comment:
Not sure this makes sense as the default height is 34px, so maybe we should just change that. Just tossing this out as a thought.